MapsAndPlacesDemo/MapsAndPlacesDemo/MainViewController.swift

```diff
@@ -275,6 +275,7 @@ class GoogleDemoApplicationsMainViewController:
         for a in actions {
             actionSheet.addAction(a as! MDCActionSheetAction)
         }
+        refreshScreen()
```

Comment:
I'm noticing `refreshScreen` can be called multiple times, for example, the handler provided in `independence` calls `refreshScreen`. Should that be removed since this method will call `refreshScreen`?

Reply:
Although there are no downsides to calling refreshScreen multiple times, I guess the first call in the handler isn't necessary. I believe the call in the handler didn't do anything because when it's called, the action menu hasn't been set yet, so it pretty much only updated the previous version of the action menu.

Should I remove the first refreshScreen?

Reply:
> Should I remove the first refreshScreen?

Yes. Please check your code to see if there are any other instances of duplication like this and try to remove it. There are downsides to calling `refreshScreen` multiple times since that method creates large objects (like view controllers). You can use the memory profiler tool in Xcode to see the difference.

Reply:
Oh I see. Ok I'll do that later today!

Reply:
Ok turns out the refreshScreen()'s weren't necessary triggers for the action menu items. I got rid of them and consolidated to just one refreshScreen() call at the end of the action menu set up function.